src/odin/enhancedtrippath.cc

```diff
@@ -915,8 +929,10 @@ std::string EnhancedTripLeg_Edge::TurnLanesToString() const {
     }
 
     // Output if marked as active
-    if (turn_lane.is_active()) {
+    if (turn_lane.state() == ::valhalla::TurnLane::kActive) {
       str += " ACTIVE";
```

Comment:
should we add in the active direction string? so it can be part of the validation for both active and valid

Reply:
Yes, thats a good idea. And that reminds me that I need to update tests to check for active_dirrection attribute! (sorry totally forgot about it)